test/rest_api/test_failure_detector.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import re
+
+def test_failure_detector_endpoints(rest_api):
+    resp = rest_api.send('GET', "failure_detector/endpoints")
+    resp.raise_for_status()
+    assert len(resp.json()) == 1
+    addr = resp.json()[0]['addrs']
+    re.match(r'\d+\.\d+\.\d+\.\d+', addr)
```

Comment:
Result of match() is ignored, so the statement has no effect.

Reply:
fixed in v2